### ISSUE TITLE
Spanish Tilde Characters not showing in Notepad++ 

### ISSUE THREAD

**Issue by estebanglas** [microsoft/cascadia-code#52]
# Environment

```none
Cascadia Code version number: 1909.16
Application (with version) used to display text: Notepad++ v 7.2.2
OS platform and version: Windows 10 64bi Pro 1903 build 18975 (Insider Preview) and 18362
Screen resolution (i.e. 220dpi): 

Any other software?
```

# Steps to reproduce
Copy or enter a spanish special character (áéíóúÁÉÍÓÚÜüñÑ) in Notepad++
Instead of showing the special Character it displays a rectangular box placeholder character

# Expected behavior

Characters should display correctly. 

# Actual behavior

Characters are not shown, rectangular placeholder is inserted instead

Comparison in Notepad++ and Windows Notepad
![image](https://user-images.githubusercontent.com/3653865/65241662-81534100-daba-11e9-89b6-d15748be9660.png)

**Comment by Aokromes** [comment]
https://github.com/microsoft/cascadia-code/issues/33 ?

**Comment by estebanglas** [comment]
Correct. @Aokromes Closing this one down. 
Should be fixed with #33
